docs/src/pages/guides/batfish-helpers.md

```diff
@@ -116,22 +112,26 @@ module.exports = () => {
 
 ```js
 import React from 'react';
-import topics from '@mapbox/batfish/data/topics';
+import filters from '@mapbox/batfish/data/filters.js';
 
 class PageShell extends React.Component {
   render() {
     return (
-      <PageLayout topics={topics} />;
+      <PageLayout filters={filters} />;
     )
   }
 }
 ```
 
 ### Output
 
-- The shape of topics is an object, where the top-level keys are pathnames for top-level pages that have subpages with `topics` or `topic`.
-  - Each object has a `topics`. It contains a unique list of topics, ordered by count of pages with that topic.
-    - Each topic has `pages`. It contains metadata for each page that has that topic.
+- The shape of filters is an object, where the top-level keys are pathnames for `examplesIndex` pages.
+  - Each object has a `products` array, a unique list of products.
+  - Each object has a `topics` array, a unique list of topics.
+  - Each object has a `pages` array of all sub pages to display. The `pages` array is ordered by the `order` frontMatter property and then alphabetically by `title`.
```

Comment:
Is this still true given the `pageSorter` function in the filter helper?

Reply:
It's not, I'll update!